packages/react/src/components/radio/RadioSelect.tsx

```diff
@@ -7,22 +7,61 @@ import {ToggleForm} from '../childForm/ToggleForm';
 import {Radio} from './Radio';
 
 export interface IRadioSelectOnChangeCallback {
+    /**
+     * A callback function executed when the selected Radio option changes
+     *
+     * @param value The value of the selected option
+     * @param id The unique identifier of the radio select
+     * @param e The change event
+     */
     onChange?: (value: string, id?: string, e?: React.MouseEvent<HTMLElement>) => void;
 }
 
 export interface IRadioSelectProps extends IRadioSelectOnChangeCallback {
+    /**
+     * A unique identifier used to indentify the RadioSelect in the PlasmaState
+     */
     id?: string;
+    /**
+     * See [input's name attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#attr-name)

```

Comment:
```suggestion
     * See [input's name attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#attr-name)
```

Hyperlink doesn't seem to be working here 🤔 
![CleanShot 2022-03-21 at 14 12 10](https://user-images.githubusercontent.com/66333175/159337436-f93b0f02-9385-4d27-ae3b-3038705ffbe5.png)

Reply:
Indeed, I'll create a task to make it work. It does work when viewing the jsdocs from vscode thought